flutter/shell/platform/tizen/tizen_renderer_egl.cc

```diff
@@ -179,12 +166,50 @@ bool TizenRendererEgl::ChooseEGLConfiguration() {
 
   EGLConfig* configs = (EGLConfig*)calloc(config_size, sizeof(EGLConfig));
   EGLint num_config;
-  if (!eglChooseConfig(egl_display_, config_attribs, configs, config_size,
-                       &num_config)) {
-    free(configs);
-    PrintEGLError();
-    FT_LOG(Error) << "No matching configurations found.";
-    return false;
+  if (enable_impeller_) {
+    EGLint impeller_config_attribs[] = {
+        // clang-format off
+      EGL_SURFACE_TYPE,    EGL_WINDOW_BIT,
```

Comment:
nit: Additional indentation
```suggestion
        EGL_SURFACE_TYPE,    EGL_WINDOW_BIT,
```

Reply:
done!